source/wp-content/themes/wporg-developer-2023/templates/single-blocks-handbook.html

```diff
@@ -0,0 +1,85 @@
+<!-- wp:template-part {"slug":"header","className":"has-display-contents"} /-->
+
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained"},"style":{"spacing":{"padding":{"left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space"}}}} -->
+<main class="wp-block-group alignfull" style="padding-left:var(--wp--preset--spacing--edge-space);padding-right:var(--wp--preset--spacing--edge-space)">
+
+	<!-- wp:group {"layout":{"type":"constrained","justifyContent":"left"},"align":"wide"} -->
+	<div class="wp-block-group alignwide">
+
+		<!-- wp:group {"tagName":"article"} -->
+		<article class="wp-block-group">
+			<!-- wp:post-title {"level":1,"style":{"spacing":{"margin":{"top":"var:preset|spacing|edge-space","bottom":"var:preset|spacing|40"}}}} /-->
+
+			<!-- wp:group {"tagName":"aside","className":"sidebar-container"} -->
+			<aside class="wp-block-group sidebar-container">
+				<!-- wp:search {"label":"Search","showLabel":false,"widthUnit":"px","buttonText":"Search","buttonPosition":"button-inside","buttonUseIcon":true,"className":"is-style-secondary-search-control","borderColor":"light-grey-1","placeholder":"Search the documentation"} /-->
+
+				<!-- wp:wporg/table-of-contents {"style":{"elements":{"link":{"color":{"text":"var:preset|color|blueberry-1"}}},"spacing":{"padding":{"top":"var:preset|spacing|20","right":"var:preset|spacing|20","bottom":"var:preset|spacing|20","left":"var:preset|spacing|20"}}},"backgroundColor":"blueberry-4","textColor":"blueberry-1"} /-->
+
+				<!-- wp:paragraph {"fontSize":"small","className":"is-link-to-top"} -->
+				<p class="has-small-font-size is-link-to-top"><a href="#wp--skip-link--target">↑ Back to top</a></p>
```

Comment:
We'll want this to be translated.

Reply:
I've moved everything into a pattern and wrapped all these in translation functions. This content isn't synced from anywhere, so it's all been manually wrapped in i18n functions.